sale_order_import/wizard/import_sale.py

```diff
@@ -0,0 +1,502 @@
+                                        'partner_shipping_id' : addr['delivery'],
+#                                         'payment_term': partner_data['value']['payment_term'], # odoo11
+                                        'payment_term':  partner_data.property_payment_term_id and partner_data.property_payment_term_id.id or False,
+                                        'order_policy' :  order_policy,
+                                        'picking_policy': picking_policy,
+#                                         'currency_id' : pricelist_data['value']['currency_id'], # odoo11
+                                        'note': row[notes].strip()
+                                        }
+            if not error_log_id:
+                error_log_id = self.env['error.log'].create({'input_file': ir_attachment.id,
+                                                             'import_user_id' : self.env.user.id,
+                                                             'import_date': datetime.now(),
+                                                             'state': 'done',
+                                                             'model_id': model.id}).id
+                                                                
+                for item in order_item_dict:
+                    order_id = self._get_order_id(order_dict[item], item, error_log_id)
+                    
+                    for so_line in order_item_dict[item]:
+                        orderline_id = self._get_orderline_id(so_line, order_id)
+
+#                     order_id.signal_workflow('order_confirm') # odoo11
+                    order_id.action_confirm() # odoo11
+                    if order_id.picking_ids:
+                        for picking in order_id.picking_ids:
+                            available = picking.action_assign()
+#                     if order_id.order_policy == 'picking': # odoo11 TODO
+                    if self.order_policy == 'picking': # TODO check now order policy removed from sale order it is on product
+                        pass
+                        #IF INVOICE POLICY IS FROM DELIVERY ORDER THEN WE WILL NOT CREATE INVOICE FROM HERE AND WE WILL PROCESS/CREATE INVOICE WHILE IMPORTING PICKINGS CSV. MEANS PICKING WILL BE HAVING STATE 2BINVOICED WILL BE PROCESS INVOICE/PAYMENT WHILE IMPORTING PICKING CSV.
+                        #invoice = picking.action_invoice_create(
+                        #                        journal_id = self.customer_invoice_journal_id.id,
+                        #                        type = 'out_invoice'
+                        #                        )
+
+#                     if order_id.state == 'manual' or order_id.state == 'prepaid':
+#                         invoice = order_id.signal_workflow('manual_invoice') # odoo 11
+
+                    invoice_ids = order_id.action_invoice_create() # odoo11
```

Comment:
The import is not success since qty_to_invoice in all sale.order.line of the sales order are always 0.0.

Reply:
I have checked the details. After setting the invoice policy of the products, the import is successful.